Zhuk.University.Tachka.Web/Pages/About.cshtml

```diff
@@ -0,0 +1,24 @@
+﻿@page
+@model Zhuk.University.Tachka.Web.Pages.AboutModel
+@{
+    ViewData["Title"] = "About Creator";
+
+}
+<div class="modal modal-sheet position-static d-block py-4" tabindex="-1" role="dialog" id="modalSheet">
+    <div class="modal-dialog" role="document">
+        <div class="modal-content rounded-4 shadow">
+            <div class="modal-header border-bottom-0">
+                <h1 class="modal-title fs-5">About me</h1>
+            </div>
+            <div class="modal-body py-0">
+                <p>First Name: Oleksandr</p>
+                <p>Last Name: Zhuk</p>
+                <p>Email: zuks01123@gmail.com</p>
```

Comment:
You can add a link here, [**like this**](https://www.tutorialspoint.com/html/html_email_links.htm), to provide more information or direct users to a relevant resource.